src/components/Planning/PlanningBreadcrumbs/PlanningBreadcrumbs.tsx

```diff
@@ -46,6 +50,12 @@ const PlanningBreadcrumbs = () => {
           value={selectedSubClass.name}
         />
       )}
+      {selectedDistrict && (
+        <BreadCrumb
+          path={`${selectedMasterClass?.id}/${selectedClass?.id}/${selectedSubClass?.id}/${selectedDistrict.id}`}
```

Comment:
The only way to get a selectedClass/subClass/district is to first have their parent selected. The only way this could happen is if a class is removed from the database.